pkgs/os-specific/linux/kmscon/default.nix

```diff
@@ -33,14 +36,21 @@ stdenv.mkDerivation rec {
     pango
     pixman
     systemd
+    mesa
+    cmake
```

Comment:
Usually cmake is not recommended when using Meson+Ninja. What are you doing here?

Reply:
I believe I added that when I was initially debugging the building of the package after updating, since it mentioned cmake being missing. Will remove and test building.

Reply:
This warning message is harmless.

When Meson tries to find the dependencies described in its `meson.build`, it tries to call some programs and scripts that can help it to find them.
One of them is cmake.

It is more useful for developers working with multiple softwares; for us package writers (especially Nixpkgs, that forces you to include every dependent package), it makes little difference.

Reply:
Reading the logs, I have noticed that the error is because our meson is still outdated.

Reply:
Ahhh, I see now. Removing it now and force-pushing.